proposals/simd/SIMD.md

```diff
@@ -6,6 +6,20 @@ current popular instruction set architectures.
 
 See also [The binary encoding of SIMD instructions](BinarySIMD.md).
 
+## Motivation
+
+WebAssembly aims to take advantage of [common hardware capabilities](https://github.com/WebAssembly/design/blob/master/Portability.md#assumptions-for-efficient-execution)
+for near native speed. The motivation for this proposal is to introduce
+WebAssembly operations that map to commonly available SIMD instructions
```

Comment:
Link to Wikipedia SIMD? https://en.wikipedia.org/wiki/SIMD

Reply:
Done.